site/utils/apiUtils/index.ts

```diff
@@ -81,3 +84,8 @@ export const getPageState = <T>(errorCookie: string, schemaObject: z.ZodType<T>,
 
     return inputsProps;
 };
+
+export const getReturnPage = (req: NextApiRequest, pageName: string) => {
+    const queryParam = req.headers.referer?.split("?")[1];
```

Comment:
we are always using REVIEW_DISRUPTION_PAGE_PATH as the 2nd argument, is there ever going to be a scenario when its a different page?

Reply:
Not for this ticket but the method could be reused at a later stage

Reply:
done